site/src/components/WorkspaceScheduleForm/WorkspaceScheduleForm.tsx

```diff
@@ -210,72 +201,120 @@ export const WorkspaceScheduleForm: FC<WorkspaceScheduleFormProps> = ({
     { value: form.values.saturday, name: "saturday", label: Language.daySaturdayLabel },
   ]
 
+  const handleToggleAutoStart = async (e: ChangeEvent) => {
+    form.handleChange(e)
+    // if enabling from empty values, fill with defaults
+    if (!form.values.autoStartEnabled && !form.values.startTime) {
+      const defaults = defaultSchedule()
+      checkboxes.forEach(async ({ name }) => {
+        await form.setFieldValue(name, defaults[name as keyof AutoStartSchedule])
+      })
+      await form.setFieldValue("startTime", defaults.startTime)
+      await form.setFieldValue("timezone", defaults.timezone)
```

Comment:
We might be able to setValues here and handle these setters in an object: https://formik.org/docs/api/formik#setvalues-fields-reactsetstateaction-field-string-any--shouldvalidate-boolean--void

Reply:
Yay, thanks!